rrtmgp/kernels/mo_gas_optics_kernels.F90

```diff
@@ -504,8 +504,12 @@ subroutine compute_Planck_source(                        &
     real(wp), dimension(ngpt,     ncol), intent(out) :: sfc_src
     real(wp), dimension(ngpt,nlay,ncol), intent(out) :: lay_src
     real(wp), dimension(ngpt,nlay,ncol), intent(out) :: lev_src_inc, lev_src_dec
+
```

Comment:
All comments from OpenACC version of kernels apply here too

Reply:
Done.